src/main/resources/assets/structurize/gui/windowbuildtool.xml

```diff
@@ -57,13 +57,15 @@
             pos="320 210"
             textcolor="Black"
             texthovercolor="White"/>
-    <button id="pastenice" tooltip="$(structurize.gui.buildtool.pastenice)" source="structurize:textures/gui/buildtool/paste.png" visible="false" size="32 32" pos="360 210"/>
+    <button id="pastenice" tooltip="$(structurize.gui.buildtool.pastenice)" source="structurize:textures/gui/buildtool/paste.png" visible="false" size="32 32" pos="355 210"/>
     <buttonimage id="undoredo"
                  tooltip="$(structurize.gui.undoredo)"
                  source="structurize:textures/gui/buildtool/undoredo.png"
                  visible="true"
                  size="32 32"
-                 pos="395 210"
+                 pos="390 210"
                  textcolor="Black"
                  texthovercolor="White"/>
+
+    <button id="showinvis" tooltip="$(structurize.gui.buildtool.show)" source="minecraft:textures/item/structure_void.png" size="26 26" pos="25 210"/>
```

Comment:
In the supplied image, the icon isnt clearly visible, suggesting some colored rectangle below it (gradient or which element we use for this)

Reply:
I used the item icon of the structurize_void block for it. Would definitely not put anything behind it, else we then get a complete icon for it.

Reply:
Then a brighter icon might be better, someone could do that later tho...